atlasdb-impl-shared/src/test/java/com/palantir/atlasdb/sweep/PersistentLockManagerTest.java

```diff
@@ -249,6 +251,14 @@ public void doesNotDeadlockOnShutdownIfLockCannotBeAcquired() throws Interrupted
         manager.shutdown();
     }
 
+    @Test
+    public void testNoOpPersistentLockDoesNotThrow() {
+        PersistentLockManager noOpManager = new PersistentLockManager(new NoOpPersistentLockService(), 0L);
+        assertTrue("NoOpPersistentLockService should return true when acquiring lock",
+                noOpManager.tryAcquirePersistentLock());
```

Comment:
How about another test for `tryAcquire` twice without a `release`? This should work for `noOp`.